test/unit/cluster_resource_discovery_test.rb

```diff
@@ -6,7 +6,7 @@ class ClusterResourceDiscoveryTest < Krane::TestCase
 
   def test_fetch_resources_failure
     crd = mocked_cluster_resource_discovery(success: false)
-    assert_raises_message(Krane::FatalKubeAPIError, "Error retrieving raw path /:") do
+    assert_raises_message(Krane::FatalKubeAPIError, "Error retrieving cluster url:") do
```

Comment:
This is just an ordering issue / implementation detail, since we now make an additional kubectl call to discover what the base path is, that call fails before the 'raw path' error.